backend/app/api/endpoints/admin/task_run_monitor.py

```
@@ -0,0 +1,33 @@
+from app.schemas.admin_task_runs import TaskRunStatsResponse
+from app.services.admin_task_run_stats import get_task_run_stats
+
+router = APIRouter(prefix="/task-runs")
+
+
+@router.get("/stats", response_model=TaskRunStatsResponse)
+async def get_admin_task_run_stats(
+    hours: int = Query(24, ge=1, le=24 * 30),
+    failure_reason_limit: int = Query(10, ge=1, le=50),
+    recent_failure_limit: int = Query(20, ge=1, le=100),
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)